src/core/data-structures/curve.js

```diff
@@ -250,25 +223,17 @@ export const Curves = {
     return n;
   },
 
-  /**
-   * ease
-   */
-  EASE: bezierCurve(0.25, 0.1, 0.25, 1.0),
+  /** ease */
+  EASE: once(bezierCurve)(0.25, 0.1, 0.25, 1.0),
```

Comment:
Calling `once()` is not statically evaluable:

```suggestion
  /**
   * @param {NormTimeDef} xVal
   * @return {NormTimeDef}
   */
  EASE(xVal) {
    return Bezier.solveYValueFromXValue(xVal, 0, 0, 0.25, 0.1, 0.25, 1.0, 1, 1);
  },
```

Reply:
Fixed. Would calling `bezierCurve` have been alright since it just returns a function?

Reply:
No, because the requires you to invoke, and invocation is never static (even if it's trivial). Only declarations should be used in Core.